lib/cocoapods-pod-merge/Main.rb

```diff
@@ -490,6 +490,7 @@ def create_podspec(merged_framework_name, pods_to_merge, podspec_info, mixed_lan
       resource_bundles = podspec_info.resource_bundles
       vendored_libraries = podspec_info.vendored_libraries
       swift_versions = podspec_info.swift_versions
+      ios_deployment_target = podfile_info.platforms.find { |platform| platform.include? "ios"}.split(',')[1]
```

Comment:
What happens if the `Podfile` does not define a platform? Will this be able to get a default value?

Reply:
That's a very valid question! By default Cocoapods provide a default in case that one is not specified:

>CocoaPods provides a default deployment target if one is not specified. The current default values are 4.3 for iOS, 10.6 for OS X, 9.0 for tvOS and 2.0 for watchOS.

Nevertheless, in this specific case, we should provide a default one in the case that is not specified in the `Podfile`. Let's do that of course!

Reply:
Uhmm ran some experiments and one thing emerged. If I delete the `MergedPods` folder and run `bundle exec pod install` Cocoapods would raise an exception like the following in case the `platform` is not set in the `Podfile`:

```ruby
It is necessary to specify the platform in the Podfile if not integrating.
```

If the project was integrated it already I think we're good if the `platform` is not specified. 

The other case is when once it was integrated we comment it out the `platform` in the `Podfile` and  run `bundle exec pod install` and Cocoapods would try to set a default platform target version for the `Pods.xcodeproj` like this:

```ruby
[!] Automatically assigning platform `iOS` with version `11.0` on target `PodMergeExample` because no platform was specified. Please specify a platform for this target in your Podfile. See `https://guides.cocoapods.org/syntax/podfile.html#platform`.
```

Let me know your thoughts about it.